src/fromager/sources.py

```diff
@@ -281,6 +285,23 @@ def patch_source(ctx: context.WorkContext, source_root_dir: pathlib.Path) -> Non
             )
 
 
+def _warn_for_old_patch(
+    source_root_dir: pathlib.Path, path: str, version: Version
```

Comment:
"path" doesn't really tell us what the argument is.

```suggestion
    source_root_dir: pathlib.Path, patch_filename: str, version: Version
```